orbslam3:
  ros__parameters:
    world_frame: "map"
    odom_frame: "odom"
    odom_ref_frame: "camera_color_frame_toro"
    camera_frame: "camera_color_frame"
    use_toro: true
    # camera_optical_frame: "camera_color_optical_frame"